dbms/src/TableFunctions/TableFunctionODBC.cpp

```diff
@@ -58,7 +77,7 @@ StoragePtr TableFunctionODBC::executeImpl(const ASTPtr & ast_function, const Con
     readStringBinary(columns_info, buf);
     NamesAndTypesList columns = NamesAndTypesList::parse(columns_info);
 
-    auto result = StorageODBC::create(table_name, connection_string, "", table_name, ColumnsDescription{columns}, context);
+    auto result = StorageODBC::create(schema_name + "." + table_name, connection_string, schema_name, table_name, ColumnsDescription{columns}, context);
```

Comment:
I think we don't need to concatenate `schema_name` with `table_name` in first argument, because of this [function](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Storages/StorageODBC.cpp#L69). It will transform query and also [concatenate](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Storages/transformQueryForExternalDatabase.cpp#L70) `table_name` and `schema_name` before query execution on remote database.

Reply:
I thought it might be clearer if some failure happens.

Reply:
Which failures are you talking about? For me it seems like redundant allocation of new string and what about case when `schema_name` is empty? Is `.table_name` OK?

Reply:
Hmm, it's indeed a useless field. 
![img-2018-08-23-170503](https://user-images.githubusercontent.com/5085485/44516134-d7f31d80-a6f6-11e8-850f-e3e41b46b8e9.png)

I'll change it back.

Reply:
Offtopic:
It seems like you use emacs) Which plugin do you use for `Find references`?